app-frontend/src/app/pages/projects/edit/annotate/annotate.scss

```diff
@@ -65,49 +65,49 @@
 
 #label-upload,
 #label-import {
-    color: #fff;
+    color: $white;
     cursor: pointer;
     font-weight: 500;
     margin-bottom: 0px;
     display: inline-block;
 }
 
 .annotate-marker{
-    background: rgba(115, 143, 252, 0.8);
+    background: rgba($brand-primary, 0.8);
     border-radius: 50%;
-    border: 2px solid rgba(53, 60, 88, 1);
+    border: 2px solid rgba($shade-dark, 1);
 }
 
 .annotate-clone-marker {
-    background: rgba(229, 115, 115, 0.5);
+    background: rgba($red, 0.5);
     border-radius: 50%;
-    border: 2px solid rgba(229, 115, 115, 0.8);
+    border: 2px solid rgba($red, 0.8);
 }
 
 .annotate-highlight-marker {
-    background: rgba(229, 115, 115, 0);
+    background: rgba($red, 0);
     border-radius: 50%;
-    border: 2px solid rgba(229, 115, 115, 0.8);
+    border: 2px solid rgba($red, 0.8);
 }
 
 .annotate-hover-marker {
-    background: rgba(115, 143, 252, 0.8);
+    background: rgba($brand-primary, 0.8);
     border-radius: 50%;
-    border: 2px solid rgba(229, 115, 115, 1);
+    border: 2px solid rgba($red, 1);
 }
 
 rf-annotate-sidebar-item{
-    &.is-sidebar-item-hovered{
+    &.sidebar-item-hover{
         .list-group-item.list-sidebar-drawn-annotations{
             border: 2px solid $brand-primary;
         }
     }
-    &.is-sidebar-item-not-hovered{
+    &.sidebar-item-not-hover{
         .list-group-item.list-sidebar-drawn-annotations{
-            border: 1px solid #ddd;
+            border: 1px solid $gray-lightest;
         }
     }
-    &.is-sidebar-item-clicked{
+    &.sidebar-item-click{
```

Comment:
I'd probably name this selected, since that's the state that this is really referring to (click is the action, selected is the stated)
the hover class is fine